src/scss/_print.scss

```diff
@@ -25,6 +25,7 @@
 	.o-crossword {
 		padding: 0!important;
 		display: block;
+		background-color: transparent!important;
```

Comment:
Something's gone really wrong with the print version of the grid. most cell borders are hidden. It might need a solid white background instead of transparent.